app/src/main/java/ch/epfl/sdp/db/DB.kt

```diff
@@ -0,0 +1,8 @@
+package ch.epfl.sdp.db
+
+import ch.epfl.sdp.game.data.Game
+
+typealias Callback<T> = (T) -> Unit
```

Comment:
This may be not the best place to declare the callback type as it is very generic.
I have, however no better file to suggest... Maybe simply in a common file ? I know this is not a best practice but this IS clearly common

Reply:
well the callback is linked to how the db works so I don't see a better place.